packages/eslint-config-base/style.js

```diff
@@ -603,7 +601,7 @@ module.exports = {
 
     // Ensure that there are no useless path segments in import statement path
     'import/no-useless-path-segments': ['warn', {
-      noUselessIndex: true,
+      noUselessIndex: false,
```

Comment:
Why did we disable this...? What's the point of using `import stuff from './module/index` when you can do `import stuff from './module'` or `import stuff from './module/index.cjs'`? 🤔

Reply:
The idea was to be more explicit.

Reply:
The primary use case was importing from sibling index which looked like `import stuff from '.'` and instead would look nicer as `import stuff from './index'`

Reply:
🤮 but 🤷‍♂️